ArduPlane/radio.cpp

```diff
@@ -285,7 +285,7 @@ void Plane::control_failsafe()
         }
     }
 
-    if(g.throttle_fs_enabled == 0) {
+    if (g.throttle_fs_enabled != 1) {
```

Comment:
Can we get an enum of the meanings for these? This is rather opaque at the moment.

Reply:
done